### PR TITLE
Invoice column in credit adjustment report

### PR DESCRIPTION
**Pull request by nickpell**
exposes more accounting data in the UI

### CODE REVIEW

**Comment by nickpell** [comment]
@benrudolph @proteusvacuum cc: @NoahCarnahan

File: corehq/apps/accounting/interface.py [hunk 1]
```diff
@@ -779,14 +786,9 @@ def rows(self):
                     "data-target": adjust_href,
                     "class": "btn",
                 }
-                columns.extend([
+                columns.append(
                     mark_safe(make_anchor_tag(adjust_href, adjust_name, adjust_attrs)),
-                    mark_safe(make_anchor_tag(
```
Comment (benrudolph) [inline corehq/apps/accounting/interface.py]: do we no longer need this?

**Comment by benrudolph** [comment]
:hammer:. one question, but then :+1:

File: corehq/apps/accounting/interface.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/accounting/interface.py]: The "Invoice" column includes a link to the same page by clicking on the invoice id hyperlink.

Example from prod:
screen shot 2015-11-30 at 11 43 09 am

I like this over the ugly button that you only see when you scroll off the screen

Comment (benrudolph) [inline corehq/apps/accounting/interface.py]: right, thanks. agreed

**Pull request merged**